src/TestingUtils/Microsoft.AspNetCore.Testing/src/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -9,10 +9,9 @@
     <!-- This is actually a library for test projects, not a test project. -->
     <IsUnitTestProject>false</IsUnitTestProject>
     <IsPackable>true</IsPackable>
-    <HasReferenceAssembly>true</HasReferenceAssembly>
-    <GenerateFrameworkReferenceAssembly>true</GenerateFrameworkReferenceAssembly>
     <!-- This package is internal, so we don't generate a package baseline. Always build against the latest dependencies. -->
     <UseLatestPackageReferences>true</UseLatestPackageReferences>
+    <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
```

Comment:
FYI the need to disable `$(IsProjectReferenceProvider)` i.e. to do more than turn `$(HasReferenceAssembly)` off will go away once this repo picks up the ref/-related changes from dotnet/aspnetcore.